web/src/features/prompts/server/prompt-router.ts

```diff
@@ -292,6 +368,44 @@ export const promptRouter = createTRPCRouter({
         throw e;
       }
     }),
+  updateTags: protectedProjectProcedure
+    .input(
+      z.object({
+        projectId: z.string(),
+        name: z.string(),
+        tags: z.array(z.string()),
+      }),
+    )
+    .mutation(async ({ input, ctx }) => {
+      throwIfNoAccess({
+        session: ctx.session,
+        projectId: input.projectId,
+        scope: "objects:tag",
+      });
+      try {
+        await auditLog({
+          session: ctx.session,
+          resourceType: "prompt",
+          resourceId: input.name,
+          action: "updateTags",
+          after: input.tags,
+        });
+        const updatedPrompts = await ctx.prisma.prompt.updateMany({
+          where: {
+            name: input.name,
+            projectId: input.projectId,
+          },
+          data: {
+            tags: {
+              set: input.tags,
+            },
+          },
+        });
+        return updatedPrompts;
```

Comment:
I would not return anything here, run risk of large return value that is not necessary